### PR TITLE
DOCS-3292: import developers

### PR DESCRIPTION
**Pull request by Sarah-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/developer/wrappers-and-sdks.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+category: 62962df622e99600810c117d
+order: 10
+---
+
+## PHP SDK
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/PHP.svg" width="100" align ="right"/>
+
+The PHP SDK for the MultiSafepay REST API.
+
+[View it on GitHub](https://github.com/MultiSafepay/php-sdk)
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:12]:
```suggestion
[View on GitHub](https://github.com/MultiSafepay/php-sdk).
```

File: content/developer/wrappers-and-sdks.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+___
+
+## Laravel
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Laravel.svg" width="100" align ="right"/>
+
+A Laravel wrapper which wraps the MultiSafepay PHP SDK.
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:19]:
```suggestion
A Laravel wrapper for MultiSafepay's PHP SDK.
```

File: content/developer/wrappers-and-sdks.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+[View it on GitHub](https://github.com/MultiSafepay/laravel-api)
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:21]:
```suggestion
[View on GitHub](https://github.com/MultiSafepay/laravel-api).
```

File: content/developer/wrappers-and-sdks.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+The official Node wrapper for the MultiSafepay API.
+
+[View it on GitHub](https://github.com/MultiSafepay/multisafepay-node-wrapper)
+
+___
+
+## Python
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Python.svg" width="100" align ="right"/>
+
+A wrapper for Python to easily use the MultiSafepay API in your Python projects.
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:46]:
```suggestion
The Python wrapper for MultiSafepay's REST API.
```

File: content/developer/wrappers-and-sdks.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+[View it on GitHub](https://github.com/MultiSafepay/multisafepay-python-wrapper)
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:48]:
```suggestion
[View on GitHub](https://github.com/MultiSafepay/multisafepay-python-wrapper).
```

File: content/developer/wrappers-and-sdks.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+___
+
+## .Net
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/NET.svg" width="100" align ="right"/>
+
+The .Net wrapper for the MultiSafepay payments API. Provides support for all payment methods.
+
+[View it on GitHub](https://github.com/MultiSafepay/.Net)
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:57]:
```suggestion
[View on GitHub](https://github.com/MultiSafepay/.Net).
```

File: content/developer/wrappers-and-sdks.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+___
+
+## Django
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Django.svg" width="100" align ="right"/>
+
+A Django integration developed by Edoburu.
+
+[View it on GitHub](https://github.com/edoburu/django-multisafepay)
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:66]:
```suggestion
[View on GitHub](https://github.com/edoburu/django-multisafepay).
```

File: content/developer/wrappers-and-sdks.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+___
+
+## GO
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Go.svg" width="100" align ="right"/>
+
+Go wrapper for the MultiSafepay API developed by kurt-stolle.
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:73]:
```suggestion
GO wrapper for MultiSafepay's REST API developed by kurt-stolle.
```

File: content/developer/wrappers-and-sdks.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+[View it on GitHub](https://github.com/kurt-stolle/go-multisafepay)
```
Comment (FrancescaNCook) [inline content/developer/wrappers-and-sdks.md:75]:
```suggestion
[View on GitHub](https://github.com/kurt-stolle/go-multisafepay).
```

**Review by FrancescaNCook — APPROVED** [review]
Sorry, slipped in a quick tech edit. Don't think I've ever looked at these pages!

File: content/developer/fastcheckout-android-sdk.md [hunk 10]
```diff
@@ -1,11 +1,21 @@
...
+title : "FastCheckout Android SDK"
+category: 62962df622e99600810c117d
+order: 70
+hidden: true
 ---
 
-As an integrator, you only need to provide a valid MSP transaction identifier or create a transaction. The FastCheckout SDK leverages the checkout process for you, providing notifications for all possible outcomes (success, pending, cancelled, etc.) once it completes a transaction. The checkout process flow includes:
+The FastCheckout Android SDK helps you connect to MultiSafepay from your native Android app. It enables a fast, frictionless, native checkout experience by storing and reusing data.
+
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Fastcheckout_Android.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/fastcheckout-android-sdk.md:10]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Fastcheckout_Android.svg" width="100" align ="right"/>
```

File: content/developer/fastcheckout-ios-sdk.md [hunk 11]
```diff
@@ -1,11 +1,18 @@
...
+title : "FastCheckout iOS SDK"
+category: 62962df622e99600810c117d
+order: 80
+hidden: true
 ---
 
-### Introduction
+Simply provide a valid transaction ID or [create an order](https://docs-api.multisafepay.com/reference/createorder). The SDK generates the checkout automatically and sends status updates for each transaction. The checkout flow includes:
+
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Fastcheckout_iOS.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/fastcheckout-ios-sdk.md:10]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Fastcheckout_iOS.svg" width="100" align ="right"/>
```

File: content/developer/fastcheckout-ios-sdk.md [hunk 12]
```diff
@@ -1,11 +1,18 @@
...
+
+[View on GitHub](https://github.com/MultiSafepay/fastcheckout-ios-sdk)
+___
+
+## Manual
 
 As an integrator, you only need to provide a valid MSP transaction identifier or create a transaction. The FastCheckout SDK leverages the checkout process for you, providing notifications for all possible outcomes (success, pending, cancelled, etc.) once it completes a transaction. The checkout process flow includes:
```
Comment (Kris-MultiSafepay) [inline content/developer/fastcheckout-ios-sdk.md:17]: Isn't this duplicating line 8?

File: content/developer/wrappers-and-sdks.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
+---
+title: 'Wrappers and SDKs'
+category: 62962df622e99600810c117d
+order: 10
+---
+
+## PHP SDK
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/PHP.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:8]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/PHP.svg" width="100" align ="right"/>
```

File: content/developer/wrappers-and-sdks.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+
+The PHP SDK for MultiSafepay's REST API.
+
+[View on GitHub](https://github.com/MultiSafepay/php-sdk).
+
+___
+
+## Laravel
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Laravel.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:17]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Laravel.svg" width="100" align ="right"/>
```

File: content/developer/wrappers-and-sdks.md [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+
+A Laravel wrapper for MultiSafepay's PHP SDK.
+
+[View on GitHub](https://github.com/MultiSafepay/laravel-api).
+
+___
+
+## Java
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Java.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:26]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Java.svg" width="100" align ="right"/>
```

File: content/developer/wrappers-and-sdks.md [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+
+A Java wrapper for MultiSafepay's payments API, which supports all payment methods.
+
+[View on GitHub](https://github.com/MultiSafepay/Java).
+
+___
+
+## Node.js
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/NodeJS.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:35]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/NodeJS.svg" width="100" align ="right"/>
```

File: content/developer/wrappers-and-sdks.md [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+
+The Node wrapper for MultiSafepay's REST API.
+
+[View on GitHub](https://github.com/MultiSafepay/multisafepay-node-wrapper).
+
+___
+
+## Python
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/Python.svg" width="100" align ="right"/>
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:44]:
```suggestion
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Python.svg" width="100" align ="right"/>
```

File: content/developer/wrappers-and-sdks.md [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+
+The Python wrapper for MultiSafepay's REST API.
+
+[View on GitHub](https://github.com/MultiSafepay/multisafepay-python-wrapper).
+
+___
+
+## .Net
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:52]:
```suggestion
## .NET
```

File: content/developer/wrappers-and-sdks.md [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/NET.svg" width="100" align ="right"/>
+
+The .Net wrapper for MultiSafepay's payments API, which supports all payment methods.
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:55]:
```suggestion
The .NET wrapper for MultiSafepay's payments API, which supports all payment methods.
```

File: content/developer/wrappers-and-sdks.md [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+A Java wrapper for MultiSafepay's payments API, which supports all payment methods.
+
+[View on GitHub](https://github.com/MultiSafepay/Java).
+
+___
+
+## Node.js
+<img src="https://raw.githubusercontent.com/MultiSafepay/docs/1281f9320696f2d256d0859421ec2cfa2350e644/static/logo/Integrations/NodeJS.svg" width="100" align ="right"/>
+
+The Node wrapper for MultiSafepay's REST API.
```
Comment (Kris-MultiSafepay) [inline content/developer/wrappers-and-sdks.md:37]:
```suggestion
The Node.js wrapper for MultiSafepay's REST API.
```

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]

File: content/developer/fastcheckout-ios-sdk.md (hunk 12 above)
Comment (Sarah-MultiSafepay) [reply content/developer/fastcheckout-ios-sdk.md:17]: This is resolved in Francesca's tech edit!

**Review by Kris-MultiSafepay — APPROVED** [review]

**Pull request merged**